deepchem/models/torch_models/modular.py

```diff
@@ -386,6 +385,7 @@ def restore(  # type: ignore
         model_dir: Optional[str]
             The path to the model directory. If None, the model directory used to initialize the model will be used.
         """
+        logger.info('Restoring model')
```

Comment:
These logger changes are also reflected in your other PR. I'm fine merging them in as part of this PR since they are small